padertorch/ops/losses/loss.py

```diff
@@ -105,12 +131,19 @@ def pit_loss(estimate, target, loss_fn=torch.nn.functional.mse_loss):
             f'{estimate.size()} != {target.size()}'
         )
     candidates = []
+    filler = (slice(None),) * axis 
     for permutation in itertools.permutations(range(sources)):
-        candidates.append(loss_fn(
-            estimate[:, permutation, :],
+        candidates.append((loss_fn(
+            estimate[filler + (permutation, )],
```

Comment:
This looks wrong.
Wouldn't is be `filler[axis] = permutation` and `estimate[filler]`?

Reply:
Have a look at my previous comment, both are correct if `axis` is computed correctly